src/tests/unit/tests/bug-filing/github/github-bug-filing-service.test.tsx

```diff
@@ -74,6 +74,13 @@ describe('GithubBugFilingServiceTest', () => {
         expect(wrapper.getElement()).toMatchSnapshot();
     });
 
+    it('renderSettingsForm: no valid settings object', () => {
```

Comment:
base on the name... and looking at the tested file... should we use `isSettingsValid` instead of `isEmpty`?
file: src\bug-filing\github\github-bug-filing-service.tsx
line: 9

Reply:
line 9? I think you are talking about line 43 right? 
If so, no, we should not. Reason is, if a user types say 5 space in the text field, although this is an invalid setting, we should let the text field display 5 spaces, instead of clear the text field.

Reply:
You're correct!